.changeset/famous-hounds-sit.md

```diff
@@ -0,0 +1,6 @@
+---
+'@backstage/backend-common': minor
+---
+
+The options part of `DatabaseManager.fromConfig` now accepts an optional logger
```

Comment:
is this a breaking change? package is below version 1

Reply:
nerfed